app/cafe.py

```diff
@@ -0,0 +1,19 @@
+import datetime
+
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)

```

Comment:
According to the checklist, it's better to use parentheses when importing multiple classes/modules. Consider changing your import style to improve the code style.